### ISSUE TITLE
[Feature Request] Templating Formats for Releases

### ISSUE THREAD

**Issue by Rage-GitHub** [wabbajack-tools/WabbaBot.NET#15]
I had the idea spark in my head while copying a code block of Discord formatting to maybe potentially have the ability to use a command, such as **/template create [name]**, to allow us to create a template based off of Discord formatting, then use the command **/release [modlist] optional:[template]** or the **/previewrelease [modlist] optional:[template]** so we don't have to retype or copy and paste each time when publishing releases. This would be more of a QoL feature but would be super handy for on the fly hotfix releases and any other sort of stuff.

**Comment by tr4wzified** [comment]
Could be possible, and then pre-filling the modal text (where you type the release message) based on the created template, right?

**Comment by Rage-GitHub** [comment]
Yes, so essentially how I have it setup for my modlist is like so:
```
*x new save is required for this update*

**Version**
> Placeholder

Placeholder Description

>>> [Download](https://www.nexusmods.com/fallout4/mods/54871?tab=files)
[Documentation](https://moddinglounge.com/aw-documentation/)
[Changelogs](https://moddinglounge.com/aw-changelogs/)
[Discord](https://discord.gg/fEBJYFnKsp)
```

Then I just add in or replace the placeholder spots that have text changed. So essentially, you'd copy and paste this into a **/template create** command and then you can pull that text up again with **/release [modlist] optional:[template]** and it'll pull up the text you put into that template. And maybe if you ever wanted to, you can go back and edit or remove templates with further commands like **/template edit [template]** or **/template remove [template]**.

**Comment by tr4wzified** [comment]
Sounds like a better alternative to #11, I'll try and think of something soon(:tm:)
